src/main/groovy/com/yodle/vantage/GenerateVantageJsonTask.groovy

```diff
@@ -48,6 +57,8 @@ class GenerateVantageJsonTask extends DefaultTask implements ReportProvider
   {
     def vantage = project.extensions.getByType(VantageExtension)
 
+    project.configurations.removeAll{it.name in doNotResolveConfigurations}
```

Comment:
they can be filtered out or removed, if you know a better solution we can try that, but that's the one I found and tested so far that seems to work.